scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/util/DateHelper.java

```diff
@@ -603,17 +602,9 @@
 	           "Pacific/Kiritimati"};
 
 	public static Date getDateWithZoneOffset(Date dateTime, TimeZone timeZone) {
-		SimpleDateFormat localSdf = new SimpleDateFormat(DATE_TIME_FORMAT);
-
-		SimpleDateFormat sdf = new SimpleDateFormat(DATE_TIME_FORMAT);
-		sdf.setTimeZone(timeZone);
-
-		try {
-			return sdf.parse(localSdf.format(dateTime));
-		} catch (ParseException e) {
-			throw new SchedulerInternalException("Invalid date specified. date=" + dateTime, e);
-		}
+		LocalDateTime local = LocalDateTime.ofInstant(dateTime.toInstant(), ZoneId.systemDefault());
```

Comment:
Regarding Time zone issue, we had tried ZoneDateTime and other possibilities using java.time package but have not been able to get the expected date time.
The approach we used LocalDateTime  is also failing during DST the whole issue is with date class which implicitly converts the JSON date string to GMT date within the entity.

We are thinking it would be the best to save date as a string in database which has no zone attached. 

What are your thought?